Tests/test_Align_emboss.py

```diff
@@ -20,14 +20,6 @@
     ) from None
 
 
-with open("Emboss/needle.txt") as handle:
-    pair_example2 = handle.read()
-
-
-with open("Emboss/needle_overhang.txt") as handle:
-    pair_example3 = handle.read()
-
-
```

Comment:
I suspected we didn't really need these files loaded as strings, although there is something to be said for at least one StringIO based test.

Reply:
I don't think we need a test using StringIO, as that would be testing StringIO, not the parser.

Reply:
I meant using StringIO or BytesIO is fairly common, and they are slightly different from a file handle - thus to me worth testing our parsers with them just in case we fall over one of those differences.